.travis.yml

```diff
@@ -1,16 +1,10 @@
-language: cpp
-sudo: required
-
+language: generic
 services: docker
 
 before_install:
- - docker pull ubuntu
- - docker run --cidfile /tmp/id -it -d ubuntu /bin/bash
+ - docker run --cidfile /tmp/id -it -d anselmolsm/travis_ci_deps:ubuntu1604 /bin/bash
```

Comment:
@lucasdemarchi nope, it takes too much time to setup the env everytime.
The image setup is in the dockerfile used to generate the image.